addons/google_calendar/tests/test_sync_odoo2google.py

```diff
@@ -798,3 +808,34 @@ def test_allday_duplicated_first_event_in_recurrence(self):
             'extendedProperties': {'shared': {'%s_odoo_id' % self.env.cr.dbname: recurrence.id}},
             'transparency': 'opaque',
         }, timeout=3)
+
+    @patch_api
+    def test_partner_type_change(self):
+        user = self.user1
```

Comment:
Could you please add a docstring in your test explaining what it does?
It will help other devs quickly identify the testing scope in the future.

Reply:
Done